README.md

```diff
@@ -128,8 +129,8 @@ kube-capacity --pods --containers --util --output yaml
   -c, --containers                includes containers in output
       --context string            context to use for Kubernetes config
   -h, --help                      help for kube-capacity
-  -n, --namespace-labels string   labels to filter namespaces with
-      --node-labels string        labels to filter nodes with
```

Comment:
Sorry I missed this earlier, I don't think `--node-labels` should be removed here.

Reply:
Huh, whoopsie!